### DISCUSSION TITLE
API untuk Quotes Terkenal

### DISCUSSION THREAD

**Discussion by dingkai031** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#321] [General]

**Comment by farizdotid** [comment]
Kalau saya googling ada mas, tapi ini developer luar. Bisa di cek disini ya : https://quotes.rest/

Sepertinya bisa bahasa Indonesia kalau kita set parameter languange nya.

**Comment by dingkai031** [comment]
okay makasihh

**Comment by SonyPradana** [comment]
Ini juga gak kalah keren [anime-chan]( https://github.com/RocktimSaikia/anime-chan)

**Comment by fdciabdul** [comment]
saya ada mas

**Comment by lakuapik** [comment]
biasanya saya ambil dari sini mas https://forismatic.com/en/. tapi bahasa inggris. contoh: http://api.forismatic.com/api/1.0/?method=getQuote&forQuote&lang=en&format=json.

kalau mau yang indonesia, saya ada simpan beberapa disini https://github.com/lakuapik/quotes-indonesia/blob/master/raw/quotes.json

**Comment by fckveza** [comment]
https://api.vhtear.com
